packages/utils/mod.ts

```diff
@@ -148,11 +148,6 @@ export function priceToUint256(priceString: string, decimals = 18) {
   return numberToBytes(priceInSmallestUnit, { size: 32 });
 }
 
-//Since we are currently storing price as a string, convert string to bigint, then calculate the decimal point.
-export function formatUnitsFromString(price: string, decimal: number) {
```

Comment:
ah, now i see. Well yes, it's not a string anymore but it's a BigInt and needs to be formatted still.